StyleCop.Console/Program.cs

```diff
@@ -36,7 +36,9 @@ public static int Main(string[] args)
 
                 var projectPath = arguments.ProjectPath;
 
-                return ProcessFolder(settings, projectPath, searchOption);
+                var outputFile = arguments.OutputFile;

```

Comment:
Yes tested:
when
     "StyleCop.Console.exe -p "MyFolder" -s Settings.StyleCop -o"
output file do not created

when
     "StyleCop.Console.exe -p "MyFolder" -s Settings.StyleCop"
default "StyleCopViolations.xml" file created in working directory

when
     "StyleCop.Console.exe -p "MyFolder" -s Settings.StyleCop -o \TestResults\stylecop.xml"
     and folder "TestResults" do not exists
output file do not created